src/applications/vaos/reducers/newAppointment.js

```diff
@@ -23,20 +26,18 @@ const initialState = {
     typeOfCareId: '323',
   },
   facilities: {},
+  systems: null,
   pageChangeInProgress: false,
   loadingSystems: false,
-  loadingFacilities: false,
 };
 
-function getFacilities(state, typeOfCareId) {
-  return state.facilities[typeOfCareId] || [];
+function getFacilities(state, typeOfCareId, vaSystem) {
+  return state.facilities[`${typeOfCareId}_${vaSystem}`] || [];
```

Comment:
I'm storing facilities in the state with a key that's the type of care and system combined, since those are the two pieces of data the api requires. This was the easiest way I found of storing the facilities so we don't have to re-fetch them all the time.